src/resources/elements/copyToClipboardButton/copyToClipboardButton.html

```diff
@@ -1,3 +1,14 @@
 <template>
-  <i ref="button" data-tippy-content="Copy to clipboard" class="copy-to-clip-button fas fa-copy" click.trigger="copy($event)"></i>
+  <svg 
```

Comment:
why replace fontawesome here?

Reply:
the font-awsome icon is not the same as the one in the design system

Reply:
We should be consistent and use the same icon everywhere in the app.

Reply:
if we need to use this icon- then we'll better use this element. But just in case- there's a `copy.svg` file in the static folder.
I had to use the inline <SVG>, in order to be able to do CSS manipulations.